apps/daffio/src/app/docs/components/heading/heading.component.html

```diff
@@ -0,0 +1,22 @@
+<ng-content></ng-content>
+<a
+	class="daffio-docs-heading-link daffio-heading-link"
```

Comment:
Don't think both classes are needed, and the class name should reflect component selector name. `daffio-ce-heading`